lib/fs.js

```diff
@@ -923,6 +937,19 @@ function stat(path, options = { bigint: false }, callback) {
   binding.stat(pathModule.toNamespacedPath(path), options.bigint, req);
 }
 
+function statfs(path, options = { bigint: false }, callback) {
+  if (typeof options === 'function') {
+    callback = options;
+    options = {};
+  }
+
+  callback = makeStatfsCallback(callback);
```

Comment:
Why not inline the function since this is the only use of it in this file?

Reply:
Fixed